Sources/ArcGISToolkit/Components/FloatingPanel/FloatingPanel.swift

```diff
@@ -127,6 +131,14 @@ struct FloatingPanel<Content>: View where Content: View {
             .onChanged {
                 let deltaY = $0.location.y - (latestDragGesture?.location.y ?? $0.location.y)
                 let proposedHeight = height + ((isCompact ? -1 : +1) * deltaY)
+                
+                if shouldSendResign {
+                    if proposedHeight < height {
```

Comment:
Should we _always_ resign first responder, not just when the user shortens the panel but anytime the handle is dragged? In practice, whenever I tried to get in a state where I could grab the handle and make it bigger, I couldn't as displaying the keyboard always maximized the height of the panel.

Reply:
Thats a good point. Yes I think doing that should be fine.